-- Variables --

local var = "Example Variable"
local gitignore = "I wish .gitignore was a file in this folder! but it is in the gitignore folder. :("
local css = "I love CSS"
local html = "I love HTML"
local Lua = "I love coding Lua"
local js = "I love coding JavaScript"
local py = "I love coding Python"
local ts = "I love coding TypeScript"
local GitHub = "I love using GitHub repositories to make updates to Code stuff and share my files to the public"
local MD = "I love using Markdown instead of .txt"
local txt = "I love using Markdown instead of boring .txt that doesnt allow masked links, i need masked links!!"
local java = "I love coding Java. Don't mix this with JavaScript."
local ruby = "I love coding Ruby"
local swift = "I love coding Swift"
local go = "I love coding Go"
local rust = "I love coding Rust"

-- Code --

print("This is a custom string that was not put inside of the Variables category.")

-- custom text above me ^^^

print(var)

if print(var) then
    print(gitignore)
end

-- but what if..

if print(gitignore) then
    print(css)
    if print(css) then
        print(html)
        if print(html) then
            print(Lua)
        end
    end
end

-- ligma balls

if print(js) then
    print(py)
end

if print(py) then
    print(ts)
end

error("Oh shit! an error!")

if error("Oh shit! an error!") then
    print("Yay, it's fixed")
end

print(GitHub)

if print(GitHub) then
    print(MD)
end

if print(MD) then
    print(txt)
end

print("I really hate txt files. I love VBS and MD more. LMAO.")

if print("I really hate txt files. I love VBS and MD more. LMAO.") then
    print(java)
end


print(ruby)

if print(ruby) then
    print(swift)
end

if print(swift) then
    print(go)
end

if print(go) then
    print(rust)
end

print("I added 4 more languages :D")

for i = 1, 10 do
    print("This is line number " .. i)
end

print("")

for j = 1, 5 do
    print("Extra line lmao " .. j)
end

print("Done!")